pinot-broker/src/main/java/org/apache/pinot/broker/routing/adaptiveserverselector/LatencySelector.java

```diff
@@ -79,4 +81,30 @@ public List<Pair<String, Double>> fetchAllServerRankingsWithScores() {
 
     return pairList;
   }
+
+  @Override
```

Comment:
(nit) may want to add a javadoc to highlight what differentiates this and the existing API and the rationale (like discussed internally) behind adding it

Reply:
The java doc is added in the interface file. Added more details to clarify the rationale.